src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/Resources/Resources.resx

```diff
@@ -0,0 +1,141 @@
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Execution_Agent_DataCollectors_EventLog_CleanupException" xml:space="preserve">
+    <value>An exception occurred while collecting final entries from the event log '{0}': {1}</value>
+  </data>
+  <data name="Execution_Agent_DataCollectors_EventLog_ContextNotFoundException" xml:space="preserve">
+    <value>EventLogDataCollector did not find eventLogContext for DataCollectionContext: {0}</value>
+  </data>
+  <data name="Execution_Agent_DataCollectors_EventLog_EventsLostWarning" xml:space="preserve">
+    <value>The '{0}' event log may have been cleared during collection; some events may not have been collected</value>
+  </data>
+  <data name="Execution_Agent_DataCollectors_EventLog_FriendlyName" xml:space="preserve">
```

Comment:
Why is this resource string?

Reply:
This is used in creating folders for `Event Log.xml`